make/local.example

```diff
@@ -0,0 +1,29 @@
+# This file contains examples for makefile flags that can be used in Cmdstan's make/local file
+
+# Change the C++ compiler
+# CXX=clang++
+
+# Enable threading
+# STAN_THREADS=true
+
+# Enable the MPI backend (requires also setting (replace gcc with clang on Mac)
+# STAN_MPI=true
+# CXX=mpicxx
+# TBB_CXX_TYPE=gcc
+
+# Enable the OpenCL backend
+# STAN_OPENCL=true
+
+# Add flags that are forwarded to the Stan-to-C++ compiler (stanc3).
+# This example enables pedantic mode
+# STANCFLAGS+= --warn-pedantic
+
+# Enable C++ compiler and linker optimization recommended by Stan developers
+# STAN_CPP_OPTIMS=true
+
+# Remove range checks from the model for faster runtime
+# STAN_NO_RANGE_CHECKS=true
+
+# Adding other arbitrary C++ compiler flags
+# CXXFLAGS+= -march=native -mtune=native
```

Comment:
[optional] The march/mtune errors that hit rstan was due to old toolchains? Any chance using these exact things as examples leads to headache?

Reply:
Those issues are probably mostly gone, but just to be safe I will add something that is less "dangerous".

Reply:
…and maybe make a note that the range checks should only be disabled for the final production run models?

Reply:
That was added in the docs here: https://github.com/stan-dev/docs/pull/365/files#diff-2eaed6ee2580309bec34cc5ba3cbe013877710c1efb49906978c9ea9eee33147R164 

But I guess it doesnt hurt to be safe yes. Its not like anyone reads the docs first and then jumps to use stuff. Myself included :)